listenbrainz/db/dump.py

```diff
@@ -783,8 +792,19 @@ def _import_dump(archive_path, db_engine: sqlalchemy.engine.Engine,
                         current_app.logger.info(
                             'Importing data into %s table...', file_name)
                         try:
-                            cursor.copy_from(tar.extractfile(member), '%s' % file_name,
-                                             columns=tables[file_name])
+                            fields = []
+                            for column in tables[file_name]:
```

Comment:
this is duplicated on line 636, is it worth abstracting out? I guess the only difference between these two queries is that one is TO STDOUT and the other is TO STDIN?

Reply:
Refactored into a separate function.